components/prospects/CreateProspectNotes.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Button, TextareaAutosize } from "@mui/material";
+import React, { useState } from "react";
+
+
+function TextBoxActions({ prospectId, setCloseTextBox }){
```

Comment:
The file name and component name should have the same name. Also, we should make this component more general because we might use it in some other place. So, let's name it something like "TextEditor" and send the "action" function as a parameter.